src/components/Helpers/Button/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from "react";
+import css from "./Button.module.scss";
+
+interface ButtonProps {
+  className?: string;
+  color?: string;
```

Comment:
```suggestion
  color?: "grey" | "orange" | "white";
```

This helps the next guy know how many types of colors are there.

Reply:
very nice ok